lib/errors.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const getContent = (err, debug, translate) => {
+  var content = {};
```

Comment:
True, but content begins as an empty object that will of course be augmented based on the outcome of the following conditions, a very typical JS pattern, and a `const` would not be allowed.